src/cli/index.js

```diff
@@ -2,5 +2,16 @@
  * CLI goes here
  */
 const program = require('commander');
+const packages = require('../../package');
 
-console.log('Cli goes here');
+program
+  .version(packages.version)
+  .description('An application for debug React-Native App')
+  .option('-d', '--developpment', 'Developpment mode')
+  .parse(process.argv);
+
+if (program.developpment) {
+  process.env.NODE_ENV = 'developpment';
+}
+
+require('../server/httpServer');
```

Comment:
I would prefer `httpServer` to expose a `start` method that will be called here. I don't like this kind of module requiring - instant calls